app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.java

```diff
@@ -30,9 +29,12 @@ public void showParkingSpaces(int parkingSpaces) {
     @Override
     public void showParkingAlertDialog() {
         DialogFragment dialog = new ConfigureParkingDialog();
+        Context context = getContext();
         FragmentManager fragmentManager = getFragmentManager();
-        if (fragmentManager != null) {
-            dialog.show(fragmentManager, FRAGMENT_TAG);
+        if(context != null){
+            if (fragmentManager != null) {
+                dialog.show(fragmentManager, context.getString(R.string.tag_parking_view_fragment));
```

Comment:
DialogFragment dialog = new ConfigureParkingDialog();
should be inside the nested if's in case there's any null

Reply:
Done!